security/tor/src/opnsense/mvc/app/views/OPNsense/Tor/general.volt

```diff
@@ -140,6 +155,7 @@ $( document ).ready(function() {
                   <td>
                       <button data-action="add" type="button" class="btn btn-xs btn-default"><span class="fa fa-plus"></span></button>
                       <!-- <button data-action="deleteSelected" type="button" class="btn btn-xs btn-default"><span class="fa fa-trash-o"></span></button> -->
+                      <button type="button" class="btn btn-xs btn-default reload_btn"><span class="fa fa-refresh"></span> {{ lang._('Reload Service') }} <i class="reloadAct_progress"></i></button>
```

Comment:
still think this should be primary colour

if you want, you can spin fa-refresh instead of adding an extra spinner ;)

also goes for the other tree below

Reply:
@fichtner thanks for the feedback - have not thought about that this icon can spin itself.